src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -0,0 +1,40 @@
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Commands.AddDraftApprenticeship;
+using SFA.DAS.CommitmentsV2.Mapping;
+
+namespace SFA.DAS.CommitmentsV2.Api.Controllers
+{
+    [Route("api/cohorts/{cohortId}/draft-apprenticeships")]
```

Comment:
Are we doing the same routing mechanism in the Update Draft Apprenticeship. I thought all the properties were going into the body?